lib/ansible/modules/packaging/os/apt.py

```diff
@@ -722,7 +722,13 @@ def cleanup(m, purge=False, force=False, operation=None,
 
 def upgrade(m, mode="yes", force=False, default_release=None,
             use_apt_get=False,
-            dpkg_options=expand_dpkg_options(DPKG_OPTIONS)):
+            dpkg_options=expand_dpkg_options(DPKG_OPTIONS), autoremove=None):
```

Comment:
Shouldn't the default for autoremove be "False" rather than "None" ? (just nitpicking)